packages/splitter/src/useSplitter.ts

```diff
@@ -161,23 +241,34 @@ export function useSplitter({
     (event: TouchEvent) => {
       const { pageY, pageX } = event.targetTouches[0];
       const elem = separatorRef.current;
-      const clientWidth = rtl ? environment.body.clientWidth : undefined;
+      const clientWidth = xor(rtl, position === SplitterPosition.LEADS)
```

Comment:
`^` relies on integers not booleans unfortunately ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR)). So I created a helper that is native to booleans to satisfy TS.

Reply:
Integers should work here since `0` and `1` is implicitly coerced to a boolean in the ternary operation. Thanks for clarifying.

Reply:
<img width="837" alt="Screen Shot 2022-01-19 at 4 44 10 PM" src="https://user-images.githubusercontent.com/1975639/150236290-baa87030-b05d-4a89-8d5c-7a91e0028224.png">

I would agree that this would be a good thing but we would need to ignore typescript. 😞

Reply:
Ah yes, thanks for pointing that out! TypeScript always protecting us. Looks like it was covered at a core team meeting: https://github.com/microsoft/TypeScript/issues/587#issuecomment-56221432